corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -102,6 +102,13 @@ def _update_progress(table_count, item_count, items_in_table):
 
 
 def _run_fast_fixture_upload(domain, workbook, task=None):
+    """This upload should be much faster than the default _run_fixture_upload with the following trade-offs:
+
+    * Does not support any fixture ownership. All fixtures must be global
```

Comment:
Would it be feasible to assert that all fixtures are global before proceeding with this task?

Reply:
https://github.com/dimagi/commcare-hq/pull/24292/commits/c9ee796f16907b56c3ec2e3775703b8566be56d1